api4/plugin_test.go

```diff
@@ -559,7 +559,7 @@ func TestGetInstalledMarketplacePlugins(t *testing.T) {
 			BaseMarketplacePlugin: &model.BaseMarketplacePlugin{
 				HomepageURL: "https://example.com/mattermost/mattermost-plugin-nps",
 				IconData:    "https://example.com/icon.svg",
-				DownloadURL: "https://example.com/mattermost/mattermost-plugin-nps/releases/download/v1.0.3/com.mattermost.nps-1.0.3.tar.gz",
```

Comment:
I don't /think/ it's strictly necessary to change this unit test, since it's entirely unrelated to the `PLUGIN_PACKAGES`, but no harm either.